### PR TITLE
update for text effect clone

### PR DESCRIPTION
**Pull request by liamcindy**
@zilongshanren  @xiaofeng11 Please review this pr, thanks.

### CODE REVIEW

**Comment by zilongshanren** [comment]
@WenhaiLin  please review this PR, thanks.

File: cocos/ui/UIText.h [hunk 1]
```diff
@@ -294,6 +294,15 @@ class CC_GUI_DLL Text : public Widget
      */
     void disableEffect(LabelEffect effect);
 
+    bool isShadowEnabled() const;
```
Comment (zilongshanren) [inline cocos/ui/UIText.h:300]: please add API comments to these newly added APIs.

File: cocos/2d/CCLabel.h [hunk 2]
```diff
@@ -349,6 +349,15 @@ class CC_DLL Label : public Node, public LabelProtocol, public BlendProtocol
      */
     virtual void disableEffect(LabelEffect effect);
 
+    bool isShadowEnabled() const { return _shadowEnabled; }
```
Comment (zilongshanren) [inline cocos/2d/CCLabel.h:355]: please add API comments to these newly added APIs.

**Comment by zilongshanren** [comment]
@liamcindy 
Cool :+1: 
please refer to my comments and add the missing API doc string.
Thanks.

**Comment by liamcindy** [comment]
@zilongshanren Please review the newly changes, thanks.

**Pull request merged**